openedx/features/classrooms/apps.py

```diff
@@ -1,8 +1,14 @@
+"""
+Apps file for classrooms application contains configuration
+"""
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
 from django.apps import AppConfig
 
 
 class ClassroomsConfig(AppConfig):
+    """
+    A Default Config class for classrooms application
```

Comment:
`A default config`

Also can we use any other work rather than config

Reply:
Added